src/mod_auth_gssapi.c

```
@@ -1693,6 +1728,21 @@ static const char *mag_name_attrs(cmd_parms *parms, void *mconfig,
     return NULL;
 }
 
+static const char *expect_name_attrs(cmd_parms *parms, void *mconfig,
+                                     const char *w)
+{
+    struct mag_config *cfg = (struct mag_config *)mconfig;
+
+    if (!expected_name_attr_expr_check(w)) {
+        ap_log_error(APLOG_MARK, APLOG_ERR, 0, parms->server,
+                     "Invalid Name Attribute filter [%s].", w);
+        return NULL;
```

I think we should return no null here, so it won get silently ignored.